packages/create-sitecore-jss/src/bin.ts

```diff
@@ -57,7 +56,8 @@ export const getDestinations = async (args: ParsedArgs, templates: string[]) =>
   // work with node-proxy destination if needed
   const proxyApp = templates.find((template) => template.match(proxyAppMatcher));
   if (proxyApp) {
```

Comment:
We need to be more explicit in our code
It's only applicable for _xmcloud proxy_ while this naming/comments mean that it's applicable for any proxy app

Reply:
while the main goal is indeed xmcloud-proxy support, we have the chance to improve devex overall, i.e. for when react+proxy setup is needed.